control/HTTP.php

```diff
@@ -383,13 +379,16 @@ public static function add_cache_headers($body = null) {
 		foreach($cacheControlHeaders as $header => $value) {
 			if(is_null($value)) {
 				unset($cacheControlHeaders[$header]);
-			} elseif(is_bool($value) || $value === "true") {
+			} elseif((is_bool($value) && $value) || $value === "true") {
 				$cacheControlHeaders[$header] = $header;
 			} else {
 				$cacheControlHeaders[$header] = $header."=".$value;
 			}
 		}
 
+		$responseHeaders['Cache-Control'] = implode(', ', $cacheControlHeaders);
+		unset($cacheControlHeaders, $header, $value);
```

Comment:
It's not really, it's just being defensive - I find it can be quite common for people to accidentally reuse vars after loops like this.